src/com/session/GoogleServlet.java

```diff
@@ -18,7 +18,7 @@
 	public void doGet(HttpServletRequest req, HttpServletResponse resp) throws IOException, ServletException {
 		
 		System.out.println("google page");
-		resp.sendRedirect("https://accounts.google.com/o/oauth2/v2/auth?scope=https://www.googleapis.com/auth/userinfo.email&access_type=offline&include_granted_scopes=true&state=/googleresp&redirect_uri=http://localhost:8888/googleresp&response_type=code&client_id=905323826918-g2vne7tcluchca01k6giosj88hhbc3el.apps.googleusercontent.com&prompt=consent");
+		resp.sendRedirect("https://accounts.google.com/o/oauth2/v2/auth?scope=https://www.googleapis.com/auth/userinfo.email&access_type=offline&include_granted_scopes=true&state=/googleresp&redirect_uri=http://1-dot-loginsession-167805.appspot.com/googleresp&response_type=code&client_id=905323826918-g2vne7tcluchca01k6giosj88hhbc3el.apps.googleusercontent.com&prompt=consent");
```

Comment:
why static redirect uri, it should be dynamically taken from request, i.e if i sent request to localhost:8888, then it should take it as redirect uri